internal/declarative/v2/default_transforms.go

```diff
@@ -63,7 +63,7 @@ func watchedByOwnedBy(_ context.Context, obj Object, resources []*unstructured.U
 			lbls = make(map[string]string)
 		}
 		// legacy managed by value
-		lbls[shared.WatchedByLabel] = OperatorName
+		lbls[shared.WatchedByLabel] = shared.WatchedByLabelValue
```

Comment:
todo: check what this is supposed to mean

Reply:
Obsolete comment, removing it